Sources/NIO/EventLoop.swift

```diff
@@ -659,7 +670,11 @@ final public class MultiThreadedEventLoopGroup: EventLoopGroup {
     private static let threadSpecificEventLoop = ThreadSpecificVariable<SelectableEventLoop>()
 
     private let index = Atomic<Int>(value: 0)
-    private let eventLoops: [SelectableEventLoop]
+    private let _eventLoops: [SelectableEventLoop]
```

Comment:
There seems to be a convention to use names starting with `_`.  See the requirements mentioned in [this comment](https://github.com/apple/swift-nio/pull/500#discussion_r199924775). This property doesn't seem to match that requirement.